modules/meb_api/app/controllers/meb_api/v0/education_benefits_controller.rb

```diff
@@ -73,8 +72,6 @@ def submit_claim
 
         clear_saved_form(params[:form_id]) if params[:form_id]
 
-        send_confirmation_email if response.ok? && Flipper.enabled?(:form1990meb_confirmation_email)
-
```

Comment:
to confirm: by removing this line and creating a dedicated endpoint, you were able to remove `bgs/service` and also avoid the `BGS::Service is uninitialized`. Correct?

Reply:
Yes, that is correct